### PR TITLE
Merge 4.12.0 into develop

### PR DESCRIPTION
**Pull request by danielmain**
This PR merges 4.12.0 into develop branch

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,20 +4,25 @@
...
+
+### Fixes
+
 - Fixed downloaded installer being left in Downloads after latest update installs ([PR 2941](https://github.com/input-output-hk/daedalus/pull/2941))
 - Fixed incorrect amount of token sent ([PR 2962](https://github.com/input-output-hk/daedalus/pull/2962))
-- Ensured non-recommended decimal place setting alert is correctly shown ([PR 3007](https://github.com/input-output-hk/daedalus/pull/3007))
 
 ### Chores
 
-- Updated cardano-node to 1.35.1 ([PR 3012](https://github.com/input-output-hk/daedalus/pull/3012))
```
Comment (michalrus) [inline CHANGELOG.md:13]: Is this not needed?

File: default.nix [hunk 2]
```diff
@@ -76,20 +80,7 @@ let
...
-      };
-    in
-      buildNodeJs {
-        enableNpm = true;
-        version = "14.17.0";
-        sha256 = "1vf989canwcx0wdpngvkbz2x232yccp7fzs1vcbr60rijgzmpq2n";
-        patches = pkgs.lib.optional pkgs.stdenv.isDarwin (njPath + "/bypass-xcodebuild.diff");
-      };
-
+    nodejs = pkgs.nodejs-16_x;
```
Comment (michalrus) [inline default.nix:83]: I missed when that happened :open_mouth: Hmmm

Comment (michalrus) [reply default.nix:83]: `release/4.12.0` has an older version – the red fragment from `develop` is correct, IMO

File: CHANGELOG.md (hunk 1 above)
Comment (danielmain) [reply CHANGELOG.md:13]: Thank you, I added it back

**Review by michalrus — APPROVED** [review]
Thank you!

**Pull request merged**